server/model/feed.go

```diff
@@ -17,21 +17,21 @@ package model
 
 // Feed represents an item in the user's feed or timeline.
 type Feed struct {
-	RepoID   int64  `json:"repo_id"                 xorm:"feed_repo_id"`
-	ID       int64  `json:"id,omitempty"            xorm:"feed_pipeline_id"`
-	Number   int64  `json:"number,omitempty"        xorm:"feed_pipeline_number"`
-	Event    string `json:"event,omitempty"         xorm:"feed_pipeline_event"`
-	Status   string `json:"status,omitempty"        xorm:"feed_pipeline_status"`
-	Created  int64  `json:"created_at,omitempty"    xorm:"feed_pipeline_created"`
-	Started  int64  `json:"started_at,omitempty"    xorm:"feed_pipeline_started"`
-	Finished int64  `json:"finished_at,omitempty"   xorm:"feed_pipeline_finished"`
-	Commit   string `json:"commit,omitempty"        xorm:"feed_pipeline_commit"`
-	Branch   string `json:"branch,omitempty"        xorm:"feed_pipeline_branch"`
-	Ref      string `json:"ref,omitempty"           xorm:"feed_pipeline_ref"`
-	Refspec  string `json:"refspec,omitempty"       xorm:"feed_pipeline_refspec"`
-	Title    string `json:"title,omitempty"         xorm:"feed_pipeline_title"`
-	Message  string `json:"message,omitempty"       xorm:"feed_pipeline_message"`
-	Author   string `json:"author,omitempty"        xorm:"feed_pipeline_author"`
-	Avatar   string `json:"author_avatar,omitempty" xorm:"feed_pipeline_avatar"`
-	Email    string `json:"author_email,omitempty"  xorm:"feed_pipeline_email"`
+	RepoID   int64  `json:"repo_id"                 xorm:"repo_id"`
```

Comment:
be aware dragons lying here ... this is a "virtual table" created by a select statement

Reply:
I've compared most parts side-by-side and could find anything in the first place